### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Echomo-Xinyu**
Java CI banner, README content has been adapted from the previous AB3 to the Teacher's Pet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 * The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
     * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
     * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
+* It is named `Teacher's Pet` (*tp* for short) because because it was designed to target the needs of teachers.
```
Comment (gnahzeus) [inline README.md:12]: Seems to be a typo here with two 'because'?

**Review by gnahzeus — COMMENTED** [review]
Minor issues, otherwise LGTM!

**Review by cadencjk — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 * The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
     * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
     * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
+* It is named `Teacher's Pet` (`tp` for short) because it was designed to target the needs of teachers.
```
Comment (gnahzeus) [inline README.md:12]: Perhaps the abbreviation 'tp' can be changed to 'TP' instead?

**Review by gnahzeus — COMMENTED** [review]
LGTM, except for one small detail!

**Pull request merged**